.ci/Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {label "clx-host-21514||clx-host-21518"}
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr:'10'))
+        timeout(time: 3, unit: 'MINUTES')
+        disableConcurrentBuilds()
+    }
+
+    environment {
+        SRC_ROOT = "${env.WORKSPACE}"
+        ARTEFACT_DIR = "${env.WORKSPACE}/artefacts"
+        NFS_WORKSPACE = "/.autodirect/mtrswgwork/_ml_devops_team/workspace/ml-nccl-rdma-sharp-plugins-pr/${env.BUILD_NUMBER}"
+        CUDA_HOME = "/hpc/local/oss/cuda10.0"
+        NCCL_DIR = "/.autodirect/mtrswgwork/artemry/ci_tools_do_not_remove/nccl-sharp-stable-2019-08-16/build"
```

Comment:
from security reasons best not to encode lab specific path into jenkinsfile. use module inside script. (if currently some staff resides inside specific dir (why? need to fix) - create nccl-env module file and load it.

Reply:
Fixed, NCCL is now built within NCCL-SHARP plugin CI - no hardcodes in `NCCL_DIR` variable now.
`NFS_WORKSPACE` is changed to `NFS_WORKSPACE = "${NFS_WORKSPACE_ROOT}/ml-nccl-rdma-sharp-plugins-pr/${BUILD_NUMBER}"` and `NFS_WORKSPACE_ROOT` is set via Jenkins UI as a string parameter. Moving this variable into a module file seems will complicate the Jenkinsfile (need to load the module within each stage or make other manipulations to pass the vars).